crates/common/certificate/src/parse_root_certificate.rs

```diff
@@ -1,6 +1,14 @@
-use rustls::{Certificate, ClientConfig, PrivateKey, RootCertStore};
-use rustls_pemfile::{certs, pkcs8_private_keys, rsa_private_keys};
-use std::{fs, fs::File, io::BufReader, path::PathBuf};
+use rustls::Certificate;
+use rustls::ClientConfig;
```

Comment:
If I remember correctly, Didier was not for this kind of import but for the old type, grouped imports. Need to confirm with @didier-wenzek

Reply:
Yes we can wait for confirmation from @didier-wenzek himself but my notes were for this method solely due to the benefits of cleaner git diffs (to assist code reviews)

Reply:
We agreed to use [`Item`](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#Item%5C%3A) and @didier-wenzek was also in the call.

Reply:
Item is perfect!